dev-system/support/run-kondo.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# **************************************************************************************************
+# This script is meant to be run from the CMR project root by Github.com as a workflow action step.
+# The purpouse of this script is to run all the linter checks using clj-kondo in a docker image and
+# to return a failed error code if there are any errors.
+#
+# This script is designed to be called from inside a docker image by github action step like this:
+#     run: ./dev-system/support/run-kondo.sh >> report.log
+
+# error or warning
+fail_level=error
+
+# true or anything but true
```

Comment:
unsure what this comment means

Reply:
bash does not have true and false, just strings, so "true" is the only thing it checks for, but I'll clear that up.